### ISSUE TITLE
Implement standard cursor types in OpenGL

### ISSUE THREAD

**Issue by benfry** [processing/processing#3554]
Hopefully we can get these from the OS or from Java. Right now, it prints a warning saying that you have to provide an image, but that's not a good option for something that's part of the API.

**Comment by benfry** [comment]
Our supported cursor types:
```
  static final int ARROW = Cursor.DEFAULT_CURSOR;
  static final int CROSS = Cursor.CROSSHAIR_CURSOR;
  static final int HAND  = Cursor.HAND_CURSOR;
  static final int MOVE  = Cursor.MOVE_CURSOR;
  static final int TEXT  = Cursor.TEXT_CURSOR;
  static final int WAIT  = Cursor.WAIT_CURSOR;
```

**Comment by codeanticode** [comment]
The implementation of https://github.com/processing/processing/issues/3769 might be useful for this.

**Comment by benfry** [comment]
Just implemented for 3.0 beta 6.

@JakubValtar can you send me the attribution information so I can add a README?
